metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/resolver/ClasspathFunctionResolver.java

```diff
@@ -65,24 +72,48 @@
  *
  */
 public class ClasspathFunctionResolver extends BaseFunctionResolver {
```

Comment:
I thought about that, but decided against it.  This is in essence still a ClasspathFunctionResolver, just one that can take alternative classloaders.  Without specifying the classloader, this functions just as it did before.